documentation/review-criteria.md

```diff
@@ -0,0 +1,89 @@
+
+## Static analysis
+There are certain issues in spec files that make it impossible to generate working SDK code or usable SDKs, CLIs or documentation. These go beyond JSON or Swagger schema correctness, as a Swagger file could correctly describe an API, yet still generate a bad SDK.
+
+We will apply these validation rules to your spec when you submit your PR. Some rules can be automatically caught with our [VS Code extension](https://github.com/Azure/openapi-lint-extension), and we highly recommend you use this tool to catch errors. Other rules need to be checked by hand both by the spec author and the reviewer.
+
+**Note there currently an exception to many of these rules for existing specs, since they were written before guidance for these rules. If you are updating an existing spec with a small change (e.g. adding a feature), make sure to mention this in your PR. We will work with your team to get your existing spec to follow all of these rules in the future, as this exception is temporary.** 
+
+### Current list of rules that are automatically caught
+These pages describe each rule in detail and how to fix the issue. They also show the effect that each issue has on the generated code, to make it easier to understand the rational behind the rule.
+- [AnonymousParameterTypes](https://github.com/Azure/autorest/wiki/anonymous-parameter-types)
+- [AvoidNestedProperties](https://github.com/Azure/autorest/wiki/avoid-nested-properties)
+- [DefaultInEnum](https://github.com/Azure/autorest/wiki/default-in-enum)
+- [DescriptiveDescriptionRequired](https://github.com/Azure/autorest/wiki/descriptive-description-required)
+- [ModelTypeIncomplete](https://github.com/Azure/autorest/wiki/model-type-incomplete)
+- [NonEmptyClientName](https://github.com/Azure/autorest/wiki/non-empty-client-name)
+- [OperationDescriptionRequired](https://github.com/Azure/autorest/wiki/operation-description-required)
+- [OperationIdNounInVerb](https://github.com/Azure/autorest/wiki/operation-id-noun-in-verb)
+- [OperationIdSingleUnderscore](https://github.com/Azure/autorest/wiki/operation-id-single-underscore)
+- [ParameterDescriptionRequired](https://github.com/Azure/autorest/wiki/parameter-description-required)
+- [RequiredPropertiesMustExist](https://github.com/Azure/autorest/wiki/required-properties-must-exist)
+- [ResponseRequired](https://github.com/Azure/autorest/wiki/response-required)
+- [ValidFormats](https://github.com/Azure/autorest/wiki/valid-formats)
+- [XmsPathsInPath](https://github.com/Azure/autorest/wiki/xms-paths-in-path)
+
+If you would like to validate your spec with these rules before submitting a PR, please use the [VS Code extension](https://github.com/Azure/openapi-lint-extension) that we provide. It will underline the file wherever there are errors.
+
+### Current list of rules that are hand-checked
```

Comment:
Good point, I'll reorganize this. It think the swagger-good-patterns could be reorganized to make it easier to find the patterns, but that'll be my next check-in. Doesn't make sense for me to duplicate the content.